code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -82,6 +82,35 @@
 
 	. += CONFIG_GET(number/outdated_movedelay/human_delay)
 
+/mob/living/carbon/human/Move()
+	. = ..()
+	Process_SL_Locator()
```

Comment:
StonedMC is in. This is not kosher and should be handled through a proper subsystem now.